jha/src/components/enrolment/ReviewTableList.vue

```diff
@@ -142,16 +173,62 @@ export default {
   },
   data: () => {
     return {
+      isApplyingForMSP: false,
       isApplyingForFPCare: false,
       isApplyingForSuppBen: false,
+      isFPCDataLoading: false,
+      isFPCDataUnavailable: false,
+      deductibleTiers: [],
+      pre1939DeductibleTiers: [],
     }
   },
-  created() {
+  async created() {
     this.isApplyingForMSP = this.$store.state.enrolmentModule.isApplyingForMSP;
     this.isApplyingForFPCare = this.$store.state.enrolmentModule.isApplyingForFPCare;
     this.isApplyingForSuppBen = this.$store.state.enrolmentModule.isApplyingForSuppBen;
+
+    let apiData = this.$store.state.appModule.deductiblesAPIData;
+
+    this.isFPCDataLoading = true;
+    this.isFPCDataUnavailable = false;
+
+    try {
+      if (!apiData) {
+        const response = await apiService.getDeductibles(
+          this.$store.state.enrolmentModule.captchaToken,
+          this.$store.state.enrolmentModule.fpcUuid
+        );
+        apiData = response.data;
+        if (!apiData || !apiData.assistanceLevels || !apiData.pre1939AssistanceLevels) {
+          throw new Error('response data does not include assistance levels.');
```

Comment:
Should there be some Splunk logging here?

Reply:
Done. Added Splunk logging in the `catch` block.